### PR TITLE
[Case study] GEDI -- needs partner approval

### PR DESCRIPTION
**Pull request by heyawhite**
Changes proposed in this pull request:

- Create new case study for GEDI

MUST ADDRESS: open questions from Gilberto in the doc

### CODE REVIEW

File: src/site/content/en/blog/gedi/index.md
Comment (malchata) [inline src/site/content/en/blog/gedi/index.md:34]:
```suggestion
experience on the web are vital to the company's overall prosperity.
```

Comment (malchata) [reply src/site/content/en/blog/gedi/index.md:34]: I believe our style guide prefers straight quotes (and double quotes).

File: src/site/content/en/blog/gedi/index.md
Comment (malchata) [inline src/site/content/en/blog/gedi/index.md:52]:
```suggestion
To address performance, the team first applied [critical rendering path
optimization](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/optimizing-critical-rendering-path), making sure there was no CSS or JavaScript that could slow down
```

Comment (malchata) [reply src/site/content/en/blog/gedi/index.md:52]: I know that this is from WebFundamentals, but this is a pretty good piece to link to. I'm sure it will make it in the migration to web.dev at some point (possibly when I create the performance course next year), so it will eventually get redirected as well.

File: src/site/content/en/blog/gedi/index.md
Comment (malchata) [inline src/site/content/en/blog/gedi/index.md:12]: This hero image has no corresponding alt text. I believe it can be added by adding an `alt` key to the front matter.

File: src/site/content/en/blog/gedi/index.md
Comment (malchata) [inline src/site/content/en/blog/gedi/index.md:73]: Needs more descriptive alt text.

File: src/site/content/en/blog/gedi/index.md
Comment (malchata) [inline src/site/content/en/blog/gedi/index.md:84]:
```suggestion
metrics such as [Time to First Byte](/ttfb/) (TTFB) and [First Contentful Paint](/fcp/) (FCP). The team used
```

File: src/site/content/en/blog/gedi/index.md
Comment (malchata) [inline src/site/content/en/blog/gedi/index.md:114]:
```suggestion
(CLS) and a 26% improvement in time to [Largest Contentful Paint](/lcp/)
```

File: src/site/content/en/blog/gedi/index.md
Comment (malchata) [inline src/site/content/en/blog/gedi/index.md:12]: Per [our media guidelines](https://web.dev/handbook/markup-media/), hero images should be 3200x960. This is too small.

**Review by malchata — CHANGES_REQUESTED** [review]
Minor edits, mostly, but enough to do here that it I'm going to require changes before merging.

File: src/site/content/en/blog/gedi/index.md
Comment (heyawhite) [reply src/site/content/en/blog/gedi/index.md:12]: That's so large! Okay, I'm on it.

**Comment by heyawhite** [comment]
@malchata all changes should be resolved.

I'm unsure if we want [Kapwing's case study](https://github.com/GoogleChrome/web.dev/pull/6962) out first, but this doesn't have any more reason to hold that I'm aware of.

**Comment by heyawhite** [comment]
One exception, possibly -- final partner review. @rowan-m for more on that.

**Comment by malchata** [comment]
I didn't own either one of these, so I'm not sure what the particulars are. I'll trust y'all to know what to do/when to smash that merge button and subscribe.

**Review by malchata — APPROVED** [review]

**Comment by gilbertococchi** [comment]
Hi all, thanks for publishing the draft.

I would like to ask two changes if possible:

Text below:
"The team then spent some time optimizing the site layout for stability by, for example, reserving space for images based on their aspect ratio."

should be replaced with (feel free to reword as necessary):

"The team then spent some time optimizing the site layout for stability by, for example, reserving space for images based on their aspect ratio. They also reserved space for Ads, first on the Top ad in the first viewport and later  also on the mid page one in order to assure users can start reading their article without experiencing annoying content jumps."

The final part of Next Steps has to go away ("GEDI's next step is to improve cumulative layout...") should be removed because outdated, will get some new content for Next steps ASAP so we can publish that.

**Comment by matthiasrohmer** [comment]
@heyawhite, Kapwing's case study is live and Gilberto's feedback seems to be in. Is this good to go? 🙂

**Comment by heyawhite** [comment]
Looks like there were additional changes made in the draft doc -- this should not be merged until that is sorted.

**Comment by gilbertococchi** [comment]
I am sorry but I am not confident we can land this article anymore.

Can we please close this item?

**Pull request closed without merging**
